Extras/MoreTankSwitches.cfg

```diff
@@ -0,0 +1,1192 @@
+@PART[hydrogen-*]:NEEDS[B9PartSwitch,CryoTanks,RationalResourcesSquad]:AFTER[zzz_CryoTanks]
+{
+	// The CryoTanks themselves
+	@MODULE[ModuleB9PartSwitch]:HAS[#moduleID[fuelSwitch]]
+	{
+		SUBTYPE
```

Comment:
I purposely avoided adding heavy, non-cryo fuels as options (MonoPropellant, LiquidFuel) to the CryoTanks. I will not accept this bit, sorry, but I don't mind providing them in other tanks that aren't purpose made to hold cryo substances.
> * "Who's going to want that much toxic, heavy, inefficient fuel (Mono) on their spaceship?"
> * "When you install CryoTanks, the impression I get is that you're interested in Hydrolox capacity, not LiquidFuel. There are many mods for huge LFO tanks already."

Reply:
Understandable.